Enums/HttpMethods.php

```diff
@@ -40,11 +41,12 @@ class HttpMethods {
      * @throws InvalidSqlOperationException
      *
      * @SuppressWarnings("PHPMD.StaticAccess")
+     * @SuppressWarnings("PHPMD.BooleanArgumentFlag")
      */
-    public static function ofSqlOperation($operation) {
+    public static function ofSqlOperation($operation, $patchInsert = false) {
```

Comment:
Default suggestion on boolean arguments is: Don't do this. This is one of a few cases where it totally makes sense :-D Just a note ;)